containers/src/Kokkos_DualView.hpp

```diff
@@ -1046,9 +1061,11 @@ class DualView : public ViewTraits<DataType, Properties...> {
       }
     };
 
-    constexpr bool has_execution_space = alloc_prop_input::has_execution_space;
-
-    if constexpr (has_execution_space) {
+    if constexpr (alloc_prop_input::sequential_host_init) {
+      sync<typename t_host::memory_space>();
+      resize_on_host(arg_prop);
```

Comment:
Does this work? It will call resync_device forwarding the properties. Doesn't that resent then need to do without initializing?

Reply:
`resync_device` largely ignores the properties. We only forward `WithoutInitiailzing`.